codec-native-quic/src/main/resources/META-INF/native-image/io.netty.incubator/netty-incubator-codec-native-quic/resource-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "resources":{
+  "includes":[
+    {
+      "condition":{"typeReachable":"io.netty.incubator.codec.quic.Quiche"},
+      "pattern":"\\QMETA-INF/native/libnetty_quiche_osx_x86_64.jnilib\\E"
+    },
+    {
+      "condition":{"typeReachable":"io.netty.incubator.codec.quic.Quiche"},
+      "pattern":"\\QMETA-INF/native/libnetty_quiche_linux_x86_64.so\\E"
```

Comment:
@violetagg dont we also need something for aarch64 and for windows ?

Reply:
will fix this